### PR TITLE
Adds contributing doc and updates README

### PR DESCRIPTION
**Pull request by alvincrespo**
This PR syncs documentation approach with the primary Redwood repo by creating a CONTRIBUTING.md file and updating the README with some additional info (forums, chats, issue search).

## Changes

- Creates CONTRIBUTING.md
- Moves Contributing and Local Development sections from README to CONTRIBUTING.md
- Updates README with "good first issue" search, forums and discord links

[README.md Rendered](https://github.com/alvincrespo/redwoodjs.com/blob/ac/add-contributing-doc/README.md)
[CONTRIBUTING.md Rendered](https://github.com/alvincrespo/redwoodjs.com/blob/ac/add-contributing-doc/CONTRIBUTING.md)

### CODE REVIEW

**Comment by alvincrespo** [comment]
Hi all! Just wanted to see if there was any feedback I could take care of on this PR?

**Comment by cannikin** [comment]
@thedavidprice Does this still work with the new contributors list in the README of the main repo?

**Comment by thedavidprice** [comment]
Ah, sorry this one slipped under my radar, gents. I'll loop back with comments tomorrow.

Thanks for pinging us @alvincrespo!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+**Table of Contents**
+
+- [Contributing](#contributing)
+  - [Local Development](#local-development)
+  - [Submitting a Pull Request](#submitting-a-pull-request)
+
```
Comment (thedavidprice) [inline CONTRIBUTING.md:8]: Add copy here at the top with links to docs to get started, i.e. Read these first:
- Code of conduct: https://github.com/redwoodjs/redwood/blob/main/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct
- Contributing: https://redwoodjs.com/docs/contributing#contributing-docs

Maybe this should be something like `## Before you start`

Comment (thedavidprice) [reply CONTRIBUTING.md:8]: We do have specific information about Docs: setup/structure and how to write. I'd like to see a specific section (maybe `## Contributing Docs`) that calls this out and links to:
https://redwoodjs.com/docs/contributing#contributing-docs

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+## Local Development
+
+This codebase is built with https://cameronjs.com and relies on plain HTML pages and Javascript with a couple helpers built in to abstract things like partials and layouts. We use https://stimulusjs.org for the few sprinkles of interactivity throughout the site.
+
+Copy `.env.example` to a file named `.env` and get API keys/IDs for all of the service listed. Alternatively, if you don't care about populating search comment out the `require` statement for `search.js` in `lib/build.js`, the `publishSearch()` function call towards the end of the `main()` function, and the `getObjectIDs()` function call at the beginning of the `main()` function.
```
Comment (thedavidprice) [inline CONTRIBUTING.md:13]: The second sentence here no longer applies and should be removed --> starting with "Alternatively..." and through the end of the paragraph. In a recent PR, a fix was added for local dev to run fine even when Env Vars are not available. So maybe add that as an explanation.

File: README.md [hunk 3]
```diff
@@ -1,35 +1,19 @@
-# redwoodjs.com
+# Redwood Website
 
 This is the repo for https://redwoodjs.com
 
-The content for the tutorial is pulled from TUTORIAL.md in this repo.
+The content for the tutorial is pulled from [TUTORIAL.md](./TUTORIAL.md) in this repo.
 
 Other documentation is pulled from various READMEs in the main [redwoodjs/redwood](https://github.com/redwoodjs/redwood) repo (see `lib/build.js`, the `SECTIONS` constant).
```
Comment (thedavidprice) [inline README.md:7]: This isn't up to date and probably doesn't belong here. Try this:
- move this info to the new `## Contributing Docs` section of CONTRIBUTING.md
- all docs are in `docs/` (and, for those that are not, there's a placeholder with link to the location, which does not need to be explained)
- the Tutorial is /TUTORIAL.md

File: README.md [hunk 4]
```diff
@@ -1,35 +1,19 @@
...
-
-If you are already already running a `yarn dev` process when you `yarn build` you may need to stop and start `yarn dev` to pick up the new pages properly.
-
 ## Contributing
 
-Open a PR against the repo on GitHub. That will build and launch a copy of the site that you can get from the `netlify/redwoodjs/deploy-preview` check (click "Details" to open it):
+Welcome and thank you for helping!
 
-![image](https://user-images.githubusercontent.com/300/76569613-c4421000-6470-11ea-8223-eb98504e6994.png)
+If this is your first time contributing - please check out our issues labeled ["good first issue"](https://github.com/redwoodjs/redwoodjs.com/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22). If you're looking for help - check out our [forums](https://community.redwoodjs.com/) or the #help channel in the [Redwood Discord](https://discord.gg/jjSYEQd).
```
Comment (thedavidprice) [inline README.md:14]: This should be moved to the intro section of /CONTRIBUTING.md

File: README.md [hunk 5]
```diff
@@ -1,35 +1,19 @@
...
 
-Double check that your changes look good!
+Check out [CONTRIBUTING.md](./CONTRIBUTING.md) for additional instructions on how to get started and submit a PR.
```
Comment (thedavidprice) [inline README.md:15]:
```suggestion
Check out [CONTRIBUTING.md](./CONTRIBUTING.md) for instructions to get started and submit a PR.
```

**Review by thedavidprice — CHANGES_REQUESTED** [review]

**Comment by thedavidprice** [comment]
@alvincrespo I (finally) got to this review. In general, breaking the content into 2 docs makes a ton of sense. However, there's just a lot of information that needs to be changed or is missing (a lot has been modified since this was written). So the review comments I made will help make this a lot more useful + accurate. Thank you!

File: CONTRIBUTING.md (hunk 1 above)
Comment (thedavidprice) [reply CONTRIBUTING.md:8]: Just remembered one more thing we need to add, project Leads. This helps new contributors know who to reach out to for help and Reviews. See this for a simple example: https://github.com/redwoodjs/redwood/tree/main/tasks/all-contributors#Package-Lead

There are three people who are primarily responsible for Reviews and merging PRs:
- `@cannikin`: website, Tutorial, and Docs
- `@jtoar`: Docs
- `@thedavidprice`: Docs

**Pull request closed without merging**